lib/utils/transpile-es6.js

```diff
@@ -40,12 +40,36 @@ module.exports = function(tree, description, _options) {
     options.plugins = options.plugins.slice();
```

Comment:
right now there is also `options.plugins` which is part of the `WIP` in the title 😉 

I think that `options.plugins` should probably be appended to the first pass plugins